bincode-transport/src/lib.rs

```diff
@@ -12,14 +12,7 @@ use async_bincode::{AsyncBincodeStream, AsyncDestination};
 use futures::{compat::*, prelude::*, ready};
 use pin_project::pin_project;
 use serde::{Deserialize, Serialize};
-use std::{
-    error::Error,
-    io,
-    marker::PhantomData,
-    net::SocketAddr,
-    pin::Pin,
-    task::{Context, Poll},
-};
+use std::{error::Error, io, marker::PhantomData, net::SocketAddr, pin::Pin, task::*};
```

Comment:
Below you do `task::{Context, Poll}` with no glob import. Shall we stick with one or the other?

Reply:
I pruned the imports and settled on `futures::task::*`.